<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormJoinGame</class>
 <widget class="QWidget" name="FormJoinGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTextEdit{	
	padding: 4px 2px 0px 8px;
	color: rgb(255, 255, 255);
}
QWidget:disabled{
	color: rgb(120, 120,120);
}
QPushButton:disabled{
	border-image: url(:/stylesheet/button_disabled.png);
}
/*PushButton*/
QPushButton{
	color: rgb(180, 188, 142);
	border-image: url(:/stylesheet/button.png);
}
QPushButton:hover, QPushButton:focus{	
	color: rgb(255, 255, 255);	
	border-image: url(:/stylesheet/button_hover.png);
}
QPushButton:pressed{
	padding-top: 2px;
	border-image: url(:/stylesheet/button_pressed.png);
}

QLabel{
	color: rgb(255, 255, 255);
}
#_l_terrain{
	border: 1px solid rgb(50, 50, 50);
	border-radius: 2px;
	background-color: rgba(255, 255, 255, 5);
}

QScrollBar:vertical {
    border: 1px solid grey;
	background-color: rgba(170, 170, 127, 100);
	border-radius:2px;
    width: 20px;
    margin: 22px 0 22px 0;
  }
  QScrollBar::handle:vertical {
	image: url(:/stylesheet/slider_handle.png);
	width: 15 px;
	height: 15 px;
  }

  QScrollBar::add-line:vertical {
	border: 1px solid rgb(170, 170, 127);
	color: rgba(170, 170, 127, 100);
	border-radius:2px;
	height: 20px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
  }

  QScrollBar::sub-line:vertical {
	border: 1px solid rgb(170, 170, 127);
	color: rgba(170, 170, 127, 100);
	border-radius:2px;
	height: 20px;
	subcontrol-position: top;
	subcontrol-origin: margin;
  }

  QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
	width: 15px;
	height: 15px;
	image: url(:/stylesheet/down_arrow.png);	
  }

  QScrollBar::up-arrow:vertical{
	image: url(:/stylesheet/up_arrow.png);	
  }

  QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
      background: none;
  }
QListView{	
	color: rgb(255, 255, 255);	
	padding:8px;
	border: 1px solid rgb(50, 50, 50);
	border-radius: 2px;
	background-color: rgba(255, 255, 255, 5);
}
QListView::item:selected{	
	color: rgb(255, 255, 255);
	background-color:rgb(255, 255, 255);
}
QListView::item:selected, QListView::item:selected:hover{
	background-color: rgba(180, 190, 140, 40);
}
QListView::item:hover{
	background-color: rgba(180, 190, 140, 20);
}
#_w_title{	
	background-color: rgba(0, 0, 0, 80);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="_w_title" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <spacer name="horizontalSpacer_19">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../resources/resources.qrc">:/stylesheet/join_game.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_20">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources/resources.qrc">:/menu/menu_top.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Revue BT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Seleccione el juego</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QListView" name="_lv_games">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>186</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Revue BT</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>220</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>220</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources/resources.qrc">:/menu/menu_middle_vertical.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="_l_terrain_text">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Revue BT</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Terreno</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="_l_terrain">
         <property name="minimumSize">
          <size>
           <width>144</width>
           <height>144</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>144</width>
           <height>144</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Revue BT</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="minimumSize">
            <size>
             <width>21</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>21</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../resources/resources.qrc">:/slots/green_team.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="_l_counter_green">
           <property name="minimumSize">
            <size>
             <width>14</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>14</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Revue BT</family>
            </font>
           </property>
           <property name="text">
            <string>9</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="minimumSize">
            <size>
             <width>21</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>21</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../resources/resources.qrc">:/slots/red_team.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="_l_counter_red">
           <property name="minimumSize">
            <size>
             <width>14</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>14</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Revue BT</family>
            </font>
           </property>
           <property name="text">
            <string>9</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_pb_accept">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Revue BT</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Aceptar</string>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_pb_cancel">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Revue BT</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources/resources.qrc">:/menu/menu_bottom.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>46</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
